keyboards/kyria/keymaps/asapjockey/keymap.c

```diff
@@ -0,0 +1,315 @@
+    ),
+// /*
+//  * Layer template
+//  *
+//  * ,-------------------------------------------.                              ,-------------------------------------------.
+//  * |        |      |      |      |      |      |                              |      |      |      |      |      |        |
+//  * |--------+------+------+------+------+------|                              |------+------+------+------+------+--------|
+//  * |        |      |      |      |      |      |                              |      |      |      |      |      |        |
+//  * |--------+------+------+------+------+------+-------------.  ,-------------+------+------+------+------+------+--------|
+//  * |        |      |      |      |      |      |      |      |  |      |      |      |      |      |      |      |        |
+//  * `----------------------+------+------+------+------+------|  |------+------+------+------+------+----------------------'
+//  *                        |      |      |      |      |      |  |      |      |      |      |      |
+//  *                        |      |      |      |      |      |  |      |      |      |      |      |
+//  *                        `----------------------------------'  `----------------------------------'
+//  */
+//     [_LAYERINDEX] = LAYOUT(
+//       _______, _______, _______, _______, _______, _______,                                     _______, _______, _______, _______, _______, _______,
+//       _______, _______, _______, _______, _______, _______,                                     _______, _______, _______, _______, _______, _______,
+//       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+//                                  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+//     ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case KC_LBR: {
+            if (record->event.pressed) {
+                if (keyboard_report->mods & MODS_SHIFT_MASK) {
+                    register_code(KC_9);
+                } else {
+                    register_code(KC_LBRC);
+                }
+            } else { // Release the key
+                if (keyboard_report->mods & MODS_SHIFT_MASK) {
+                    unregister_code(KC_9);
+                } else {
+                    unregister_code(KC_LBRC);
+                }
+            }
+            return false;
+            break;
+        }
+        case KC_RBR: {
+            if (record->event.pressed) {
+                if (keyboard_report->mods & MODS_SHIFT_MASK) {
+                    register_code(KC_0);
+                } else {
+                    register_code(KC_RBRC);
+                }
+            } else { // Release the key
+                if (keyboard_report->mods & MODS_SHIFT_MASK) {
```

Comment:
```suggestion
                if ((get_mods() & MODS_SHIFT_MASK) == MOD_MASK_SHIFT) {
```

Reply:
It should be MODS_MASK_SHIFT in the bit-wise and right?